src/utils.ts

```diff
@@ -28,7 +28,7 @@ export enum CatalogRecordItems {
 
 export const addRasterJobActions = (job: Job): void => {
   if (job.domain === CatalogRecordItems.RASTER) {
-    const isRestorable = job.type === 'Ingestion_New' || job.type === 'Ingestion_Update'; // && !!job.parameters && job.parameters.ingestionResolution;
+    const isRestorable = job.type === 'Ingestion_New' || job.type === 'Ingestion_Update'; // Important: job.parameters === null, getJobs API excludes parameters field
```

Comment:
hardcoded values 'Ingestion_New' and 'Ingestion_Update' are a bit problematic but for now let's leave it in this way